fedimint-bitcoind/src/bitcoincore_rpc.rs

```diff
@@ -358,7 +358,10 @@ pub struct EsploraClient(esplora_client::AsyncClient);
 
 impl EsploraClient {
     fn new(url: &Url) -> anyhow::Result<Self> {
-        let builder = esplora_client::Builder::new(url.as_str());
+        // Url needs to have any trailing path including '/' removed
+        let without_trailing = url.origin().ascii_serialization();
```

Comment:
This will remove the path component of the URL altogether which can be problematic if the API is accessible at a non-root path. E.g. blockstream's testnet API:

```
https://blockstream.info/testnet/api/tx/<txid>/hex
```

Reply:
Fixed this, looks like we have a markdown check failure on master though.